common/src/test/kotlin/streams/utils/SchemaUtilsTest.kt

```diff
@@ -20,6 +20,48 @@ class SchemaUtilsTest {
         assertEquals(setOf("foo"), keys)
     }
 
+    @Test
+    fun `getNodeKeys should select (with multiple labels) on of the constraints with lowest properties and, with same size, the first label in alphabetical order and, with same label name, the first sorted properties list alphabetically`() {
```

Comment:
ecco magari mettiamo un nome più semplice `getNodeKeys should return the key sorted properly` e aggiungi un commento nel body del metodo su cosa ci aspettiamo :D